Requirements.txt

```diff
@@ -0,0 +1,32 @@
+Functionele eisen:
+
+    De applicatie moet verschillende gebruikersrollen ondersteunen, waaronder beheerders (Hella, Paulus, Robin) en personeelsleden, met elk hun specifieke rechten en toegangsniveaus.
+    Beheerders hebben toegang tot alle functies die de applicatie biedt.	
+    Personeelsleden kunnen alle gegevens m.b.t. productinformatie en voorraad inzien, maar niet wijzigen.
+    De applicatie moet de mogelijkheid bieden om de voorraad van producten (tv's, afstandsbedieningen, muurbeugels, CI-modules) bij te houden, zowel inkomend als uitgaand.
+    Robin moet de mogelijkheid hebben om in de applicatie de locatie van producten in het magazijn bij te houden en te beheren.
```

Comment:
welke rol heeft robin hier? zolas gezegd beter in rollen dan in namen te praten.